AsyncAwait.Task2.CodeReviewChallenge/Extensions/LoggingRequestMiddlewareExtensions.cs

```diff
@@ -1,12 +0,0 @@
-﻿using AsyncAwait.Task2.CodeReviewChallenge.Middleware;
```

Comment:
Since I have removed the Statistic middleware, this extension method is redundant.